templates/resources/index.html

```diff
@@ -39,7 +50,7 @@ <h1>Resources</h1>
 <section class="p-strip is-shallow">
   <header class="row">
     <div class="col-12 u-align--right">
-      <form action="?foo=bar" method="get" class="p-form p-form--inline">
+      <form action="/" method="get" class="p-form p-form--inline">
```

Comment:
This redirect to the root path. It is probably best being empty.

Reply:
Done.